manifests/install/charts/as-a-second-scheduler/templates/rbac.yaml

```diff
@@ -54,10 +57,21 @@ rules:
   resources: ["subjectaccessreviews"]
   verbs: ["create"]
 - apiGroups: ["storage.k8s.io"]
-  resources: ["csinodes", "storageclasses"]
+  resources: ["*"]
   verbs: ["get", "list", "watch"]
+# resources need to be updated with the scheduler plugins used
 - apiGroups: ["scheduling.sigs.k8s.io"]
-  resources: ["podgroups", "elasticquotas"]
+  resources: [
+  {{- if .Values.podgroups.enable }} 
```

Comment:
we may define some sub-fields under scheduler plugin name in values.yaml to indicate which CRDs are associated.